### DISCUSSION TITLE
Which Database Structure do you use for HA?

### DISCUSSION THREAD

**Discussion by scottpi** [basecamp/kamal#1261] [Q&A]
Hey, we are currently on GKE and would like to move our cluster to Kamal. But now we faced some issues with Postgres and with the HA system there. 
So I'd like to know what DB structure you use with minimal effort and a lot of resilience.

**Comment by tuladhar** [comment]
Hey @scottpi, Can you share more details about your issue with Postgres and HA system? I am not sure if I understood when you say DB structure.

**Reply by scottpi** [reply]
Yes sure, as far as I know, you have to configure the postgres-ha manually with pgpool etc..
But I would like to have a fail-safe database (preferably Postgres) across several servers without a lot of configuration effort.

And do you have for every Service a new postgres service or do you have one postgres servers with several databases inside there?
Thank you!
